app/Services/NotificationService.php

```
@@ -0,0 +1,290 @@
+
+            return false;
+        }
+    }
+
+    /**
+     * Send custom notification to a user
+     *
+     * @param  array  $customData
+     * @return bool
+     */
+    public function sendCustom(User $user, array $data, array $channels = [], $customData = [])
```

PHPMD (2.15.0)

Avoid unused parameters such as '$customData'. (Unused Code Rules)

(UnusedFormalParameter)